<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDiskGuiClass</class>
 <widget class="QWidget" name="CDiskGuiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CDiskGui</string>
  </property>
  <widget class="QPushButton" name="refreshButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>170</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
  <widget class="QPushButton" name="uploadButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>240</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>上传</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="filelistWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>511</width>
     <height>591</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>文件名</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>文件大小</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="ipEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="portBox">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>10086</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="pathEdit">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>30</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>./server_root</string>
   </property>
  </widget>
  <widget class="QLabel" name="path">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>10</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器文件路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="ip">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>60</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>服务器IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="port">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>110</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>端口号</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downloadButton">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>310</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>下载</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="cdisk_gui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>refreshButton</sender>
   <signal>clicked()</signal>
   <receiver>CDiskGuiClass</receiver>
   <slot>refresh()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1563</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>1815</x>
     <y>1125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>uploadButton</sender>
   <signal>clicked()</signal>
   <receiver>CDiskGuiClass</receiver>
   <slot>upload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>634</x>
     <y>90</y>
    </hint>
    <hint type="destinationlabel">
     <x>705</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>downloadButton</sender>
   <signal>clicked()</signal>
   <receiver>CDiskGuiClass</receiver>
   <slot>download()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>350</y>
    </hint>
    <hint type="destinationlabel">
     <x>742</x>
     <y>469</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>refresh()</slot>
  <slot>upload()</slot>
  <slot>download()</slot>
 </slots>
</ui>
